testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeRedirect.java

```diff
@@ -877,6 +877,30 @@ public void testViewReferencingHiveAndDeltaTable(boolean legacyHiveViewTranslati
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testHiveToDeltaPropertiesRedirect()
+    {
+        String tableName = "test_redirect_to_delta_properties_" + randomNameSuffix();
+
+        onDelta().executeQuery("" +
+                "CREATE TABLE " + tableName + " USING DELTA " +
+                "LOCATION '" + locationForTable(tableName) + "' " +
+                " AS SELECT true AS a_boolean");
+
+        List<Row> expected = List.of(
+                row("delta.minReaderVersion", "1"),
+                row("delta.minWriterVersion", "2"));
+
+        try {
+            assertThat(onTrino().executeQuery(format("SELECT * FROM delta.default.\"%s$properties\"", tableName))).containsOnly(expected);
+            assertThat(onTrino().executeQuery(format("SELECT * FROM hive.default.\"%s$properties\"", tableName))).containsOnly(expected);
+        }
+        finally {
+            onTrino().executeQuery("DROP TABLE hive.default." + tableName);
```

Comment:
Drop the table on Delta because we created it in Delta.

Reply:
Reminder.

Reply:
I believe the diff is outdated here. It is changed to `delta.default`.

Reply:
I mean using `onDelta`.

Reply:
Ah, I see, thank you. I'll fix it